src/main/java/duke/datetime/DateTimeConverter.java

```diff
@@ -0,0 +1,56 @@
+package duke.datetime;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class DateTimeConverter {
+    protected String[] inputSplit;
+
+    /**
+     * Constructs a new DateTimeConverter object to convert date and time Strings to LocalDate and LocalTime.
+     *
+     * @param inputSplit  String input that has been split into description, date, from time and to time.
+     */
+    public DateTimeConverter(String[] inputSplit) {
+        this.inputSplit = inputSplit;
+    }
+
+    /**
+     * Returns LocalDate object converted from user inputted date String of format dd-MM-yyyy.
+     *
+     * @return LocalDate created from inputted date.
+     */
+    public LocalDate convertDate() {
+        String date = inputSplit[1].substring(3);
+
+        if (date.length() > 10) {
+            date = date.substring(0, date.length() - 1);
+        }
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+        LocalDate formattedDate = LocalDate.parse(date, formatter);
+        return formattedDate;
+    }
+
+    /**
+     * Returns LocalTime object converted from user inputted time of format h AM/PM.
+     *
+     * @param s  Type of time, start (indicated by "from") or end (indicated by "to") time.
+     * @return LocalTime created from input time.
+     */
+    public LocalTime convertTime(String s) {
```

Comment:
Would a more descriptive variable name be more suitable since the function parameter has a large scope?